.github/workflows/ci.yaml

```diff
@@ -49,32 +56,26 @@ jobs:
           target: wasm32-unknown-unknown
           components: rust-src
       - name: Rust Cache
-        uses: Swatinem/rust-cache@v2
+        uses: Swatinem/rust-cache@v2.7.3
       - name: Configure Git
         run: git config --global url."https://${{ secrets.GH_READ_TOKEN }}:x-oauth-basic@github.com/".insteadOf "https://github.com/"
-      - name: Install try-runtime
-        run: cargo install --git https://github.com/paritytech/try-runtime-cli --tag v0.8.0 --locked
-      - name: Check Build
-        run: |
-          cargo build --release --features try-runtime
-      - name: Check Try-Runtime
+      - name: Check Core Components
         run: |
-           try-runtime --runtime ./target/release/wbuild/cere-runtime/cere_runtime.compact.compressed.wasm \
```

Comment:
Could you explain why you removed the `try-runtime` step ? It's important to ensure the new runtime upgrade will be executed without errors before merging the PR